src/Simplify.elm

```diff
@@ -1910,12 +1910,40 @@ expressionVisitorHelp (Node expressionRange expression) config context =
         toCheckInfo :
             { fnRange : Range
             , fn : ( ModuleName, String )
+            , nbOfArgs : Int
```

Comment:
Adding this makes _so_ much sense!
Maybe call it `argCount`?

Reply:
Sure